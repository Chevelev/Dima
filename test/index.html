<!DOCTYPE html>
<html lang="en">
    <head>
	    <meta charset="utf-8">
	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	    <meta name="viewport" content="width=device-width, initial-scale=1">
	    <title>Bootstrap 101 Template</title>
	    <link href="css/bootstrap.css" rel="stylesheet">
	    <link href="css/style.css" rel="stylesheet">
	    <link href="css/reset.css" rel="stylesheet">
    </head>

    <body>
    	<header>
	    	<div class="container">
	    		<div class="row">
			   		<div class="col-md-4">
			   			<div class="layout-logo">Layout</div>
			   		</div>
			  		<div class="col-md-4">
			   			<nav class="navpannel">
		    				<ul>
		    					<li><a href="#">html</a></li>
		    					<li><a href="#">css</a></li>
		   						<li><a href="#">javascript</a></li>
			   				</ul>
			   			</nav>
		   			</div>
		   			<div class="col-md-4">
		    			<button class="pull-right">Enter</button>
		    		</div>
			   	</div>
			</div>
			<div class="motivate">
				<h1>Monetize your skills now!</h1>
				<button>Go!</button>
			</div>
			<div id="carousel-example-generic" class="carousel slide" data-ride="carousel" data-pause="hover">
			   	<ol class="carousel-indicators">
			    	<li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
			    	<li data-target="#carousel-example-generic" data-slide-to="1"></li>
			    	<li data-target="#carousel-example-generic" data-slide-to="2"></li>
			    	<li data-target="#carousel-example-generic" data-slide-to="3"></li>
			    	<li data-target="#carousel-example-generic" data-slide-to="4"></li>
			  	</ol>
			  	<div class="carousel-inner" role="listbox">
			    	<div class="item active">
			      		<img src="img/header-bg.jpg" alt="Photo-1">
			    	</div>
			    	<div class="item">
			      		<img src="img/header-bg.jpg" alt="Photo-2">
			    	</div>
			    	<div class="item">
			      		<img src="img/header-bg.jpg" alt="Photo-3">
			    	</div>
			    	<div class="item">
			      		<img src="img/header-bg.jpg" alt="Photo-4">
			    	</div>
			       	<div class="item">
			      		<img src="img/header-bg.jpg" alt="Photo-5">
			    	</div>
			    </div>
			  	<a class="left carousel-control" href="#carousel-example-generic" role="button" data-slide="prev">
			    	<span class="glyphicon glyphicon-menu-left" aria-hidden="true"></span>
			    	<span class="sr-only">Previous</span>
			  	</a>
			  	<a class="right carousel-control" href="#carousel-example-generic" role="button" data-slide="next">
				    <span class="glyphicon glyphicon-menu-right" aria-hidden="true"></span>
				    <span class="sr-only">Next</span>
				</a>
			</div>
		</header>

		<main>
			<section class="user-info">
				<div class="container">
					<div class="row">
						<div class="col-md-6">
							<h2>Something about you</h2>
							<p>In this section, you can specify any information about yourself, insert your photos, your interests and skills. It is also desirable to use tag canvas.<br>
							Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</p>
						</div>
						<div class="col-md-5 col-md-offset-1">
							<figure>
								<figcaption>Your photo</figcaption>
								<img src="img/user-img.jpg" alt="Your photo">	
							</figure>
						</div>
					</div>
				</div>
			</section>
			<section class="register">
				<div class="container">
					<div class="col-md-6 col-md-offset-3 form-wrapper">
						<form action="#" method="POST">
							<h3>Register</h3>
							<div class="form-group">
								<input type="email" class="form-control" placeholder="Email" id="user-email">
							</div>
							<div class="form-group">
								<input type="password" class="form-control" placeholder="Password" id="user-password">
							</div>
							<div class="form-group">
								<select class="form-control" id="country-select">
									<option value="">Country</option>
									<option value="">Country 1</option>
									<option value="">Country 2</option>
									<option value="">Country 3</option>
									<option value="">Country 4</option>
									<option value="">Country 5</option>
								</select>
							</div>
						</form>
					</div>
				</div>
			</section>
		</main>

		<footer>
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<p>Â© 2015 layout</p>
					</div>
				</div>
			</div>
		</footer>

	    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
	    <script src="js/bootstrap.min.js"></script>
    </body>
</html>