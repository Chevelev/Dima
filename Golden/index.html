<!DOCTYPE html>
<html lang="ru-UA">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Golden</title>
    <!-- Bootstrap -->
    <link href="css/bootstrap.css" rel="stylesheet">
    <link href="css/style.css" rel="stylesheet">
    <link href="css/jquery.mCustomScrollbar.css" rel="stylesheet">
    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

<body>

  <section class="header">
    <div class="container">
      <nav class="nav-bar navbar-default">
        <div class="navbar-header">
          <a class="navbar-brand" href="#"><img src="img/logo.png" height="28" width="79" alt=""/></a>
          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#responsive-menu">
            <span class="sr-only">Menu</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
        </div>
        <div class="collapse navbar-collapse" id="responsive-menu">
          <ul class="nav navbar-nav">
            <li><a href="menu-home">Home</a></li>
            <li><a href="menu-services">Services</a></li>
            <li><a href="menu-portfolio">Portfolio</a></li>
            <li><a href="menu-about">About</a></li>
            <li><a href="menu-contact">Contact</a></li>
          </ul>
        </div>
      </nav>
      <h3>Welcome To Our Studio!</h3>
      <h1>Itâ€™s nice to Meet you</h1>
      <button class="btn">Tell Me More</button>
    </div>
  </section>
      
  <section class="services">
    <div class="container">
      <h2 class="title">Services</h2>
      <p class="description">Proin iacults purus consequat sem cure.</p>
      <div class="row">
        <div class="col-xs-12 col-sm-4 icon commerce">
        	<h5 class="title">E-Commerce</h5>
          <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p> 
        </div>
        <div class="col-xs-12 col-sm-4 icon responsive">
        	<h5 class="title">Responsive Web</h5>
          <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit </p> 
        </div>
        <div class="col-xs-12 col-sm-4 icon security">
        	<h5 class="title">Web Security</h5>
          <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit</p> 
        </div>
      </div>
      </div>
    </div>
  </section>    
       
  <section class="portfolio">
    <div class="container">
      <h2 class="title">Our Portfolio</h2>
      <p class="description">Proin iacults purus consequat sem cure.</p>
      <div class="row white-cover">	
        <div class="col-xs-12 col-sm-4">
          <a href=""><img src="img/ebony.png" height="300" width="291" alt=""/></a>
          <div class="p-item-footer">
            <h5 class="title">Ebony & Ivory</h5>
            <span>Branding</span>
          </div>
        </div>
        <div class="col-xs-12 col-sm-4">
          <a href=""><img src="img/smart.png" height="300" width="291" alt=""/></a>
          <div class="p-item-footer">
            <h5 class="title">Smart Stationary</h5>
            <span>Print Design</span>
          </div>
        </div>
        <div class="col-xs-12 col-sm-4">
          <a href=""><img src="img/clever.png" height="300" width="291" alt=""/></a>
          <div class="p-item-footer">
            <h5 class="title">Clever Poster</h5>
            <span>Print Design</span>
          </div>
        </div>
        <div class="col-xs-12 col-sm-4">
          <a href=""><img src="img/vynil.png" height="300" width="291" alt=""/></a>
          <div class="p-item-footer">
            <h5 class="title">Vinyl Record</h5>
            <span>Product Mock-Up</span>
          </div>
        </div>
        <div class="col-xs-12 col-sm-4">
          <a href=""><img src="img/treehouse.png" height="300" width="291" alt=""/></a>
          <div class="p-item-footer">
            <h5 class="title">Treehouse Template</h5>
            <span>Web Design</span>
          </div>
        </div>
        <div class="col-xs-12 col-sm-4">
          <a href=""><img src="img/burned.png" height="300" width="291" alt=""/></a>
          <div class="p-item-footer">
            <h5 class="title">Burned Logo</h5>
            <span>Branding</span>
          </div>
        </div>
      </div> 
    </div>
  </section>
       
  <section class="about">
    <div class="container">
      <h2 class="title">About Us</h2>
      <p class="description">Proin iacults purus consequat sem cure.</p>
      <div class="row">
        <article class="col-xs-12">
          <div class="col-xs-6 col-sm-4 text-right">
            <h5 class="title"><span>July 2010</span><br/>Our Humble Beginning</h5>
            <p>Proin iaculis purus consequat sem cure 
               digni ssim. Donec porttitora entum suscipit 
               aenean rhoncus posuere odio in tincidunt. Proin 
               iaculis purus consequat sem cure digni 
               ssim. Donec porttitora entum suscipit.</p>
          </div>
          <div class="col-xs-6 col-sm-4 img-circle">
            <img src="img/about1.png" alt=""/>
          </div>
        </article> 

        <article class="col-xs-12">
          <div class="col-xs-6 col-sm-4 col-sm-offset-4 img-circle">
            <img src="img/about2.png" alt=""/>
          </div>
          <div class="col-xs-6 col-sm-4 text-left">
            <h5 class="title"><span>January 2011</span><br/>Facing Startup Battles</h5>
            <p>Proin iaculis purus consequat sem cure 
               digni ssim. Donec porttitora entum suscipit 
               aenean rhoncus posuere odio in tincidunt. Proin 
               iaculis purus consequat sem cure digni 
               ssim. Donec porttitora entum suscipit.</p>
          </div>
        </article> 

        <article class="col-xs-12">
          <div class="col-xs-6 col-sm-4 text-right">
            <h5 class="title"><span>December 2012</span><br/>Enter The Dark Days</h5>
            <p>Proin iaculis purus consequat sem cure 
               digni ssim. Donec porttitora entum suscipit 
               aenean rhoncus posuere odio in tincidunt. Proin  
               iaculis purus consequat sem cure digni 
               ssim. Donec porttitora entum suscipit.</p>
          </div>
          <div class="col-xs-6 col-sm-4 img-circle">
            <img src="img/about3.png" alt=""/>
          </div>
        </article> 

        <article class="col-xs-12">
          <div class="col-xs-6 col-sm-4 col-sm-offset-4 img-circle">  
            <img src="img/about4.png" alt=""/>
          </div>
          <div class="col-xs-6 col-sm-4 text-left">
            <h5 class="title"><span>February 2014</span><br/>Our Triumph</h5>
            <p>Proin iaculis purus consequat sem cure 
               digni ssim. Donec porttitora entum suscipit 
               aenean rhoncus posuere odio in tincidunt. Proin 
               iaculis purus consequat sem cure digni 
               ssim. Donec porttitora entum suscipit.</p>
          </div>
        </article>

        <article class="col-xs-12"> 
          <div class="col-xs-12 col-sm-4 col-sm-offset-4">
            <a href="#"><span class="img-circle-link">Our<br>Story<br>Continues<br>...</span></a>
          </div>
        </article> 
        
      </div>
    </div>
  </section>
       
  <section class="team">
    <div class="container">
      <h2 class="title">our amazing team</h2>
      <p class="description">Proin iacults purus consequat sem cure.</p>
      <div class="row">
        <article class="col-sm-4 col-xs-12">
          <img src="img/team1.png" height="220" width="220" alt=""/>
          <h5 class="title">Kimberly Thompson</h5>
          <p class="cat">Marketer</p> 
          <div class="social">
            <a href="#" class="ic-tw">&nbsp;</a>
            <a href="#" class="ic-fb">&nbsp;</a>
            <a href="#" class="ic-pn">&nbsp;</a>
          </div>        
        </article>
        <article class="col-sm-4 col-xs-12">
          <img src="img/team2.png" height="220" width="220" alt=""/>
          <h5 class="title">Rico Massimo</h5>
          <p class="cat">Coder</p> 
          <div class="social">
            <a href="#" class="ic-tw">&nbsp;</a>
            <a href="#" class="ic-fb">&nbsp;</a>
            <a href="#" class="ic-gl">&nbsp;</a>
          </div>        
        </article>
        <article class="col-sm-4 col-xs-12">
          <img src="img/team3.png" height="220" width="220" alt=""/>
          <h5 class="title">Uku Mason</h5>
          <p class="cat">Graphic Designer</p> 
          <div class="social">
            <a href="#" class="ic-fb">&nbsp;</a>
            <a href="#" class="ic-pn">&nbsp;</a>
          </div>        
        </article>
      </div>
      <p class="description team-info">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit <br /> 
      aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
    </div>
  </section>
       
  <section class="brand">
    <div class="container">
      <div class="row">
        <a href="#" class="col-sm-3 col-xs-12 ic-brand"><img src="img/envato.jpg" alt=""></a>
        <a href="#" class="col-sm-3 col-xs-12 ic-brand"><img src="img/wordpress.png" alt=""></a>
        <a href="#" class="col-sm-3 col-xs-12 ic-brand"><img src="img/tuts.jpg" alt=""></a>
        <a href="#" class="col-sm-3 col-xs-12 ic-brand"><img src="img/microlancer.jpg" alt=""></a>
      </div>
    </div>
  </section>
       
  <section class="contact">
    <div class="container">
      <h2 class="title">contact us</h2>
      <p class="description">Proin iacults purus consequat sem cure.</p>
      <div class="row">
        <form action="#" id="messege">
          	<div class="col-sm-6 col-xs-12">
              <input type="text" placeholder="Your Name *"/>
              <input type="text" placeholder="Your e-mail *"/>
              <input type="text" placeholder="Subject"/>
            </div>
         	  <div class="col-sm-6 col-xs-12">
              <textarea cols="0" rows="0" placeholder="Your message *"></textarea>
            </div>
          </div>
          <button class="btn">Send Message</button>
        </form>
      </div>
    </div>
  </section>
       
  <footer class="copyright">
    <div class="container">
      <div class="row">
        <div class="col-sm-4 col-xs-12 copy"><p>&copy; Copyright 2014 FreebiesXpress.com</p></div>
        <div class="col-sm-4 col-xs-12 social-ic">         
          <div class="social">  
            <a href="#" class="ic-tw black">&nbsp;</a>
            <a href="#" class="ic-fb black">&nbsp;</a>
            <a href="#" class="ic-pn black">&nbsp;</a>
            <a href="#" class="ic-gl black">&nbsp;</a>
          </div>
        </div>        
    </div>
  </footer>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
<script src="js/jquery.mCustomScrollbar.concat.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<script>
(function($){
$(window).load(function(){
 
$("body").mCustomScrollbar({
theme:"dark-thin"
});
 
});
})(jQuery);
</script>
</body>
</html>
